src/components/Jobs/index.tsx

```
@@ -1,10 +1,10 @@
-import { PeanutGuyGIF } from '@/assets'
+import { PeanutPeace } from '@/assets'
```

ESLint

[error] '`@/assets`' import is restricted from being used. Import from a specific file instead of the '`@/assets`' barrel — barrels force the bundler to load every re-export and hurt build perf. See CLAUDE.md.

(no-restricted-imports)